docs/docs/add-custom-webpack-config.md

```diff
@@ -25,9 +25,9 @@ call each build type a "stage". The following stages exist:
 4. build-html: production build static HTML pages
 5. build-javascript: production JavaScript build. Creates route bundles as well as a `commons` and `app bundle`.
 
-Check [https://github.com/gatsbyjs/gatsby/blob/1.0/packages/gatsby/src/utils/webpack.config.js](https://github.com/gatsbyjs/gatsby/blob/1.0/packages/gatsby/src/utils/webpack.config.js) for the source.
+Check [webpack.config.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/webpack.config.js) for the source.
 
```

Comment:
you are right, you mean like this `/packages/gatsby/src/utils/webpack.config.js`?